cmd/websocktunnel/main.go

```diff
@@ -28,6 +28,7 @@ Environment:
  TASKCLUSTER_PROXY_SECRET_A                  JWT secret
  TASKCLUSTER_PROXY_SECRET_B                  alternate JWT secret
  SYSLOG_ADDR                                 address to which to send syslog output
+ AUDIENCE									 JWT 'audience' claim 
```

Comment:
did the whitespace get mixed up here?

Reply:
I should fix this quick

Reply:
Looks like it's the github's tab indentation that's causing it